owncloudApp/src/main/java/com/owncloud/android/presentation/files/createfolder/CreateFolderDialogFragment.kt

```diff
@@ -82,6 +97,29 @@ class CreateFolderDialogFragment : DialogFragment() {
         alertDialog.setOnShowListener {
             val okButton = alertDialog.getButton(AlertDialog.BUTTON_POSITIVE)
             okButton.isEnabled = isButtonEnabled
+
+            okButton.setOnClickListener {
+                var fileName: String = inputText.text.toString()
+                var error: String? = null
+                if (fileName.length > MAX_FILENAME_LENGTH) {
+                    error = String.format(
+                        getString(R.string.uploader_upload_text_dialog_filename_error_length_max),
+                        MAX_FILENAME_LENGTH
+                    )
+                } else if (fileName.isEmpty()) {
+                    error = getString(R.string.uploader_upload_text_dialog_filename_error_empty)
+                } else if (pattern.containsMatchIn(fileName)) {
+                    error = getString(R.string.filename_forbidden_characters)
```

Comment:
Maybe this part and the check of `MAX_FILENAME_LENGTH` could be moved to the `TextChangedListener` for the `inputText` so that we can see the error appearing or disappearing in real time just after adding the forbidden character or exceeding the maximum length, without the need to press the OK button and thus saving one click and some time, what do you think? 🤔

Reply:
Done in the three dialogs: create a folder, create a file, and create a file when is receive from externa